releases/tools/release-2.0.0+.md

```diff
@@ -0,0 +1,23 @@
+---
+id: tools-2-0-0
+title: Multiplayer Tools 2.0.0-pre+
+description: Release notes for Multiplayer Tools including new features, updates, bug fixes, known issues, and information to help you upgrade.
+---
+
+## [2.0.0-pre.2] - 2023-05-02
```

Comment:
I think `2.0.0-pre.2` was somewhat skipped, and that this text and filename should now be `2.0.0-pre.3`, but please confirm with @fluong6 and/or @DenninDalke